### DISCUSSION TITLE
Up arrow doesn't work

### DISCUSSION THREAD

**Discussion by steambsd** [lxqt/lxqt#1961] [General]
In the previous topic, they helped me and everything works well, except for the last problem that haunts me.
I think it’s because of WineHQ - but it’s not certain.

**Comment by tsujan** [comment]
Up arrow not working is a too general description of a problem. If it doesn't work anywhere, the physical key may have been damaged. If it works in some places, it may have been used as a single shortcut and so, consumed. It may also have been mapped to another key by a script of yours. And so on....

**Reply by steambsd** [reply]
It doesn't work anywhere. But if i use numlock - arrow up work only on numpad

**Reply by tsujan** [reply]
You could boot from the live ISO of a Linux distro to make sure that the physical key is OK.

**Reply by steambsd** [reply]
Ok. But it is strange for FreeBSD. LXQT very amazing - i wont use it.

**Reply by tsujan** [reply]
The advantage of using a live ISO is that it's completely independent of your system; otherwise, you could also test the Up arrow key under a minimal Openbox session.
